xc/common/utils/prjxray_arch_import.py

```diff
@@ -742,9 +743,13 @@ def main():
     synth_loc_map = {}
     synth_tile_map = {}
     roi = None
-    if args.use_roi:
-        with open(args.use_roi) as f:
-            j = json.load(f)
+    if args.use_roi or args.partition_region:
```

Comment:
Given that partition_region and roi in this case do the example same thing, why do we need another argument?

Reply:
Yes I agree it's unnecessary right now.  I'm using it for other related work (like heterogeneous synth tiles).  My thought was to leave it in for now to prevent having to make major changes to cmake when I post a pr for that in a day or two.  But maybe that's not the best idea...

Reply:
As far as I can tell, your work is a pure superset of the original ROI concept.  I believe we should be treating it like an extension, rather than a side by side piece?